pkg/directory/directory_unix.go

```diff
@@ -1,4 +1,4 @@
-// +build linux
+// +build linux freebsd
```

Comment:
I'm not sure if _unix suffix works as expected, it's not mentioned in go documentation, so maybe _unix files should have the tags?